decidim/decidim
# Show character counter when replying to message

#### :tophat: What? Why?
When replying to conversation, text field has maximum length (default 1000 characters) but before it has not been shown to the user. Also when viewing conversation avatar image paths were broken.

#### Testing
1. User Menu -> Conversations -> My groups -> *pick group*
2. Click "New conversation" and pick some user
3. Start a conversation
4. Click "chevron-right" icon to reply to the conversation
5. Type something and see "characters left" below the text field.

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
![image](https://user-images.githubusercontent.com/19709320/155760456-24f43962-5b08-409e-b789-fd3699eb78e9.png)

:hearts: Thank you!
